job/validators.py

```diff
@@ -209,42 +209,43 @@ def cross_clean(self):
         PV tilt set to latitude if not provided and prod_factor_series validated
         """
         def cross_clean_pv(pvmodel):
-            if pvmodel.__getattribute__("tilt") == 0.537:  # 0.537 is a dummy number, default tilt
-                pvmodel.__setattr__("tilt", self.models["Site"].__getattribute__("latitude"))
+            if pvmodel.__getattribute__("tilt") == None:  # 0.537 is a dummy number, default tilt # TODO: change to None?
```

Comment:
delete comment/todo

Reply:
@hdunham just to be certain, do you happen to know if there's any reason we need to use this dummy value of 0.537 as opposed to setting the default to None as we do with other inputs?

Reply:
removed comment in https://github.com/NREL/REopt_API/pull/343/commits/066d132881ab26f9c5d9270a3700244f37e23d3c